modules/virt-setting-policy-attributes.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-schedule-vms.adoc
+
+[id="virt-setting-policy-attributes_{context}"]
+= Setting a policy attribute and CPU feature
+
+You can set a policy attribute and CPU feature for each virtual machine (VM) to ensure that it is scheduled on a node according to policy and feature. The CPU feature that you set is verified to ensure that it is supported by the host CPU or emulated by the hypervisor.
+
+.Prerequisites
+* You configured CPU node discovery.
```

Comment:
Default in CNV

Reply:
Because it is a default we can possibly reomve this line, as the user will not see the component itself.

Reply:
Removed.

Reply:
It is is still present in this PR